### PR TITLE
interfaces: drop udev tagging from framebuffer interface

### PR DESCRIPTION
**Pull request by chunsangjeong**
This will fix access denied of opengl interface when it's used with framebuffer
interface in the same snap. It's a first step of fix which allow us to unblock
the referenced case immediately and the proper udev tagging will be included in
a second step.

Fixes: https://bugs.launchpad.net/snapd/+bug/1675738
Signed-off-by: Chunsang Jeong <chunsang.jeong@canonical.com>

### CODE REVIEW

**Comment by zyga** [comment]
I have merged master into this branch, resolved conflicts resulting from new snapd APIs and pushed this back to https://github.com/zyga/snapd/tree/fix-opengl-interface-udev-tag-dev-fb. I also contacted @chunsangjeong to make sure he is aware of this but I was unable to push into the pull request myself. Once the PR is updated it should be ready for review.

File: interfaces/builtin/framebuffer.go [hunk 1]
```diff
@@ -83,17 +82,6 @@ func (iface *FramebufferInterface) AppArmorConnectedPlug(spec *apparmor.Specific
 
 // Getter for the security snippet specific to the plug
 func (iface *FramebufferInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
-	switch securitySystem {
```
Comment (femdom) [inline interfaces/builtin/framebuffer.go]: Looks better now=)

Does it mean that udev tagging will never be enabled for the framebuffer interface?

Comment (zyga) [inline interfaces/builtin/framebuffer.go]: I think we will do a pass and enable it for everything. Now the problem is that any interface using udev tagging breaks every interface not using it that needs to access devices.

Comment (morphis) [inline interfaces/builtin/framebuffer.go]: We have a work item to convert all interfaces to use proper udev tagging for our next sprint which will then land in a future snapd version 2.24/2.25/2.26 depending on complexity and timing.

**Comment by morphis** [comment]
LGTM

**Comment by chunsangjeong** [comment]
Added TODO comment and re-pulled https://github.com/zyga/snapd/tree/fix-opengl-interface-udev-tag-dev-fb. @zyga please check the updated branch.

**Comment by femdom** [comment]
Thanks! I am going to test this today in the next 8 hours.

**Comment by chunsangjeong** [comment]
@zyga I just rebased with the latest code and re-commited from there. Please check if it still conflicts with your branch.Thanks.

**Comment by jdstrand** [comment]
Approving, but please adjust framebuffer_test.go accordingly (then your tests should start passing).

**Comment by morphis** [comment]
LGTM

**Comment by arapulido** [comment]
Nice! Merged! Thanks all!

**Pull request merged**